### PR TITLE
Point to cmd installation instructions from README

### PR DESCRIPTION
**Pull request by chrisglass**
While installing the CLI, I noticed a couple of small touch-ups that could be made to the READMEs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ description: the Go library for Exoscale
 
 A wrapper for the [Exoscale public cloud](https://www.exoscale.com) API.
 
+This project also provides an executable CLI client. See [the relevant subtree](https://github.com/exoscale/egoscale/tree/master/cmd/exo) for installation instructions.
+
```
Comment (pierre-emmanuelJ) [inline README.md]: That a good point 👍

File: cmd/exo/README.md [hunk 2]
```diff
@@ -25,7 +25,12 @@ $ go install
 ## Configuration
 
 A configuration file holding your credentials is required.
-You can generate one via a guided prompt.
+You can generate one via a guided interactive prompt with:
+
+```
+$ exo config
+```
+
```
Comment (greut) [inline cmd/exo/README.md]: as it duplicates what's below under _automatic configuration_, why not removing the _You can generate..._ line?

File: README.md (hunk 1 above)
Comment (chrisglass) [reply README.md]: Ah yeah, even better!

:+1:

File: cmd/exo/README.md (hunk 2 above)
Comment (chrisglass) [reply cmd/exo/README.md]: Fair enough :)

Comment (chrisglass) [reply cmd/exo/README.md]: Actually, how about moving the "Automatic configuration" header to be "configuration"?

/me does

**Comment by greut** [comment]
the `exo config` output is outdated, feel free to remove it as well.

**Comment by greut** [comment]
thank you very much @chrisglass

**Pull request merged**